mm2src/derives/enum_from/src/from_stringify.rs

```diff
@@ -0,0 +1,90 @@
+use crate::from_trait::get_attr_meta;
+use crate::{CompileError, IdentCtx, MacroAttr};
+use itertools::Itertools;
+use proc_macro2::{Ident, Span, TokenStream};
+use quote::__private::ext::RepToTokensExt;
+use quote::{quote, ToTokens};
+use syn::__private::TokenStream2;
+use syn::spanned::Spanned;
+use syn::NestedMeta::Lit;
+use syn::{NestedMeta, Variant};
+
+pub(crate) fn check_inner_ident_type(ident: Option<Ident>) -> Result<(), CompileError> {
+    if let Some(ident) = ident {
+        if ident.to_string().as_str() == "String" {
+            return Ok(());
+        }
+    };
+
+    Err(CompileError::expected_string_inner_ident(MacroAttr::FromStringify))
+}
+
+fn get_variant_unnamed_ident(fields: syn::Fields) -> Option<Ident> {
+    if let syn::Fields::Unnamed(syn::FieldsUnnamed { unnamed, .. }) = fields {
+        if let Some(field) = unnamed.iter().next() {
+            if let Some(syn::Type::Path(type_path, ..)) = field.ty.next().cloned() {
+                let type_path = type_path.path.segments.iter().next().cloned()?.ident;
+                return Some(type_path);
+            };
+        };
+    }
+    None
+}
+
+/// The `#[from_stringify(..)]` attribute value.
+struct AttrIdentToken(TokenStream);
+
+impl TryFrom<NestedMeta> for AttrIdentToken {
+    type Error = CompileError;
+
+    /// Try to get a trait name and the method from an attribute value `Trait::method`.
+    fn try_from(attr: NestedMeta) -> Result<Self, Self::Error> {
+        match attr {
+            Lit(lit) => Ok(Self(lit.to_token_stream())),
+            _ => Err(CompileError::expected_literal_inner()),
+        }
+    }
+}
+
+fn parse_inner_ident(ident: String, span: Span) -> Result<Ident, CompileError> {
+    let ident = ident.split("").into_iter().filter(|&e| e != "\"").join("");
```

Comment:
I think the previous implementation (with `strip_prefix`, `strip_suffix`) was better in terms of performance.
`String::split` could be replaced with `String::chars`, but `Iterator::filter` iterates over each char in the string to remove `"`. However `strip_prefix` and `strip_suffix` checks the first and the last elements correspondingly.
`O(n)` vs `O(1)`

Reply:
good. I reverted the implementation..